span-processing-config-service-impl/src/main/java/org/hypertrace/span/processing/config/service/utils/TimestampConverter.java

```diff
@@ -0,0 +1,13 @@
+package org.hypertrace.span.processing.config.service.utils;
+
+import com.google.protobuf.Timestamp;
+import java.time.Instant;
+
+public class TimestampConverter {
+  public static Timestamp convert(Instant instant) {
```

Comment:
We should not have static methods. It makes it hard to unit test. You can inject the `TimestampConverter` wherever needed